packages/cypress/tests/cypress/e2e/custom-downloads-directory.cy.ts

```diff
@@ -1,7 +1,9 @@
 context('using Cypress.config', () => {
   it('Downloads archives to the user-specified folder', () => {
     cy.visit('/viewports');
-    const dirExists = cy.task('directoryExists', '/some-dir');
-    console.log('DIRECTORY EXISTS', dirExists);
+    const chromaticArchivesDir = `${Cypress.config('downloadsFolder')}/chromatic-archives`;
+    cy.task('directoryExists', chromaticArchivesDir).then((dirExists) => {
+      expect(dirExists).true;
```

Comment:
I didn't realize one could do `.true` -- is there a reason you used that here instead of `.toBe(true)`?

Reply:
No reason, changed it in f74bde795646780bdbf4fe76b331102439c7ab5d